RadioItem/tests/RadioItem-specs.js

```diff
@@ -16,31 +16,31 @@ describe('RadioItem Specs', () => {
 	});
 
 	test('should render `children`', () => {
-		const {getByText} = render(<RadioItemBase>Hello RadioItem</RadioItemBase>);
+		render(<RadioItemBase>Hello RadioItem</RadioItemBase>);
 
-		const actual = getByText('Hello RadioItem');
+		const actual = screen.getByText('Hello RadioItem');
 
 		expect(actual).toBeInTheDocument();
 	});
 
 	test('should render a disabled RadioItem when `disabled` is true', () => {
-		const {getByRole} = render(<RadioItemBase disabled>Hello RadioItem</RadioItemBase>);
-		const radioItem = getByRole('checkbox');
+		render(<RadioItemBase disabled>Hello RadioItem</RadioItemBase>);
+		const radioItem = screen.getByRole('checkbox');
 
 		expect(radioItem).toHaveAttribute('disabled');
 	});
 
 	test('should render a `slot before`', () => {
-		const {getByText} = render(<RadioItemBase slotBefore="arrowup">Hello RadioItem</RadioItemBase>);
+		render(<RadioItemBase slotBefore="arrowup">Hello RadioItem</RadioItemBase>);
 
-		const actual = getByText('arrowup');
+		const actual = screen.getByText('arrowup');
 
 		expect(actual).toBeInTheDocument();
 	});
 
 	test('should select RadioItem with click', () => {
-		const {getAllByRole} = render(<RadioItem>Hello RadioItem</RadioItem>);
-		const radioItem = getAllByRole('checkbox');
+		render(<RadioItem>Hello RadioItem</RadioItem>);
+		const radioItem = screen.getAllByRole('checkbox');
 
 		fireEvent.click(radioItem[0]);
```

Comment:
let's use userEvent here

Reply:
I tried userEvent, but for some reason it doesn't work and I don't know why :(